src/game.rs

```diff
@@ -0,0 +1,225 @@
+use std::collections::HashMap;
+
+use crate::cell::Cell;
+
+pub struct Game {
+    live_cells: Vec<Cell>,
+}
+
+impl Game {
+    pub fn new_with_seed(seed: Vec<(i32, i32)>) -> Self {
+        let live_cells = seed.iter().map(|(x, y)| Cell::new(*x, *y)).collect();
```

Comment:
```suggestion
        let live_cells = seed.into_iter().map(|(x, y)| Cell::new(x, y)).collect();
```
We can consume the vector with `into_iter` as we don't use it after, avoiding the dereference

Reply:
I like that, done!